### ISSUE TITLE
Minio, Nginx, Postgres etc.

### ISSUE THREAD

**Issue by KrishnaPG** [illacloud/deploy-illa-manually#12]
Currently the docker compose just uses one single image for hosting everything inside a single container. It is creating all dependencies (such as Minio, Ngnix, Postgres etc.) fresh inside its own container.

This makes it difficult to reuse existing Minio, Nginx or Postgres servers that are already running outside.

Please allow configuration, so that existing Minio, Nginx, Postgres (and any other dependencies) can be specified and reused for those who have them already running (without having to launch new instances inside the illa container again).

**Comment by karminski** [comment]
@KrishnaPG That's true. We are considering making anoter docker-compose slim image for it. We'll let you know when it's done.

**Comment by devMls** [comment]
Interesting It would be usefull fornme
